Symfony/CS/Fixer/PSR2/ElseifFixer.php

```diff
@@ -30,25 +30,31 @@ class ElseifFixer extends AbstractFixer
     public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
+        for ($index = 7, $count = count($tokens); $index < $count - 1; ++$index) {
```

Comment:
It is always the same reason isn't it? Do not check tokens that are no candidate anyway.
I see no readability issues here at all.

Besides the code was a simple `foreach`:
```php
 foreach ($tokens->findGivenKind(T_ELSE) as $index => $token) {
```

however at least two main contributors told me in another PR to avoid `findGivenKind` where possible....